src/safe/util/noop.ts

```diff
@@ -4,5 +4,5 @@
  * @param args - optional params
  * @returns void
  */
-function noop<T = unknown[]>(...args: T): void { return void args; } // eslint-disable-line no-void
+function noop<T = unknown[]>(...args: T[]): void { return void args; } // eslint-disable-line no-void
```

Comment:
This change is incorrect, as T defaults to unknown[], and not all arguments provided need be the same type T~